### PR TITLE
Adds details for better DELETE queries in sql-standards.md

### PR DESCRIPTION
**Pull request by Gultask**
### Description

You didn't see the typo
image

### Related Issue

Closes

## Thank you for contributing to the AzerothCore wiki.

Remember that the wiki is currently available in English and Spanish.

- https://www.azerothcore.org/wiki/home
- https://www.azerothcore.org/wiki/es/home

### CODE REVIEW

**Review by Helias — APPROVED** [review]

File: docs/sql-standards.md [hunk 1]
```diff
@@ -32,6 +32,21 @@ DELETE FROM `creature_loot_template` WHERE `entry` = 3 AND `item` = 884;
 INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES 
 (3, 884, 0, 40, 1, 1, 0, 1, 1, 'Comment');
 ```
+---
+
+When performing DELETE queries, always specify more than one field when possible.
```
Comment (Helias) [inline docs/sql-standards.md:37]:
```suggestion
When performing DELETE queries, always specify more than one field when possible, in particular, specify the primary key present in the table.
```

Comment (Helias) [reply docs/sql-standards.md:37]: saying only "multiple fields" can be misunderstood or misleading

Comment (heyitsbench) [reply docs/sql-standards.md:37]: Specifying another column/example as well as the primary key would likely be beneficial. In the case of the `creature` table, only the `guid` column is in the primary key, but the standard seems to be to specify another distinguishing column as well.

Comment (Gultask) [reply docs/sql-standards.md]: How about this? Some tables have more than one primary key
```
When performing DELETE queries, always specify more than one field. In particular, specify at least one primary key present in the table and another field.
```

**Review by Helias — APPROVED** [review]

File: docs/sql-standards.md (hunk 1 above)
Comment (Helias) [reply docs/sql-standards.md]: about `creature` table, the concept is always the same, I think it's ok now

**Pull request merged**